packages/domains-table/src/utils/can-bulk-update.ts

```diff
@@ -0,0 +1,17 @@
+import { PartialDomainData } from '@automattic/data-stores';
+import { type as domainTypes } from './constants';
+import { getDomainType } from './get-domain-type';
+
+export function canBulkUpdate( domain: PartialDomainData ): boolean {
+	if ( domain.wpcom_domain || domain.is_wpcom_staging_domain ) {
+		return false;
+	}
+
+	return ! (
+		[
+			domainTypes.SITE_REDIRECT,
```

Comment:
@p-jackson  Should this not include `domainTypes.TRANSFER`?

Reply:
Oops I never pushed that yesterday 😪